src/NetAnalyzers/RulesMissingDocumentation.md

```diff
@@ -2,3 +2,4 @@
 
 Rule ID | Missing Help Link | Title |
 --------|-------------------|-------|
+CA1840 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1840> | Provide memory-based overrides of async methods when subclassing 'Stream' |
```

Comment:
ID has already been used, please choose a new one.

Reply:
Changed ID to ca1842